HomeTask6/app/src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="footer">
+    <h5 style="float: left;margin-left: 20px;">Footer</h5>
```

Comment:
Do not use inline styles in templates
use ngClasses, ngStyles instead

Reply:
Done